### ISSUE TITLE
Potential issue with pleroma/akkoma links

### ISSUE THREAD

**Issue by fyr77** [tvler/streetpass#64]
Hi, thanks for this cool project!

I have attempted to link my own profile on my website https://notjakob.com/ in the `<head>` section, but it does not seem to be recognised by the extension. 

In my case, the link does not include the `@`, *oma links are just `https://instance.tld/username` and not `https://instance.tld/@username` like with Mastodon - could this be the source of the issue on my site?

**Comment by cphuntington97** [comment]
I found this comment because I have multiple rel="me" links on my website https://charadepodcast.com however streetpass only seems to recognize my Mastodon profile link. At first I thought it might be an issue of not recognizing more than one profile per page, which still might be true, but also, the profile that is not being recognized is Pleroma. I used the format `https://instance.tld/users/username` which Pleroma seems to use.

**Comment by DeclanChidlow** [comment]
Another Akkoma user here finding that StreetPass doesn't detect my fedi profile.

**Comment by jamescridland** [comment]
+1 from me - it doesn't detect an Akkoma fedi profile.

**Comment by foxyseta** [comment]
> In my case, the link does not include the `@`, *oma links are just `https://instance.tld/username` and not `https://instance.tld/@username` like with Mastodon - could this be the source of the issue on my site?

I do not think so. Sharkey does not work either, despite having `@` in links. Perhaps the dev is currently using the Mastodon API, as opposed to the ActivityPub API?
